tests/integrate_test/cita_charge_mode.sh

```diff
@@ -54,6 +47,7 @@ main() {
     echo -n "1) generate config  ...  "
     create_config \
         --contract_arguments "SysConfig.checkFeeBackPlatform=true" \
+        --contract_arguments "SysConfig.checkCreateContractPermission=true" \
```

Comment:
No need.
and `--contract_arguments "SysConfig.checkFeeBackPlatform=true"`

Reply:
permission setting is needless, checkFeeBackPlatform is used while checking transferd balance.

Reply:
Is not checkFeeBackFlatform used for fee_back test?

Reply:
Setting `SysConfig.checkFeeBackPlatform=true`, the transaction fee goes to chain owner,
when remove this config, tx fees goes to miner which causes original transfer test cases fail.

I do not want to change orginal test logic, so remain checkFeeBackPlatform=true.

Reply:
I see.